src/app/dashboard/import-export/export-modal/export-modal.component.ts

```diff
@@ -30,6 +31,14 @@ export class ExportModalComponent implements OnInit {
     } else {
       this.exportDirectoryData();
     }
+    if (this.noDirectoryAssessments.length == 1 && this.exportData.directories.length == 0) {
```

Comment:
you're on the right track with this solution, but when a just a folder is selected to be exported then name defaults to the name of the folder the selected folder is in when it should default to the name of the selected folder.